app/react/Map/specs/Map.spec.js

```diff
@@ -368,4 +383,23 @@ describe('Map', () => {
       expect(instance.updateMapStyle).toHaveBeenCalled();
     });
   });
+
+  describe('replaceKeysMapStyleJson', () => {
+    it('should set the map tiler key from the settings', async () => {
+      render();
+      const style = {
+        sources: { openmaptiles: { url: 'URL?key={{MAP_TILER_KEY}}' } },
+        glyphs: 'URL?key={{MAP_TILER_KEY}}',
+      };
+      const expectedKey = 'XYZ';
+      settingsAPI.get.mockResolvedValue({ mapTilerKey: expectedKey });
+      await instance.replaceKeysMapStyleJson(style);
+      const stringifyStyle = JSON.stringify(instance.mapStyle);
+      expect(stringifyStyle).toEqual(expect.not.stringContaining('{{MAP_TILER_KEY}}'));
```

Comment:
this can be written as:
`expect(stringifyStyle).not.toContain('{{MAP_TILER_KEY}}')`

Please refer to: https://jestjs.io/docs/en/expect#tocontainitem

Reply:
Great!, thanks for the reference 👍